src/uct/sm/mm/mm_iface.h

```diff
@@ -67,10 +69,19 @@ struct uct_mm_iface {
 
 
 struct uct_mm_fifo_element {
-    uint8_t             flags;
-    uint8_t             am_id;     /* active message id */
-    uint16_t            length;    /* length of actual data */
-    /* the data follows here */
+    uint8_t         flags;
+    uint8_t         am_id;          /* active message id */
+    uint16_t        length;         /* length of actual data */
+
+    /* bcopy parameters */
+    size_t          desc_mpool_size;
```

Comment:
that would imply that the size is of the desc, but it should be of the entire memory pool it belongs to